safari/app-store-description.txt

```diff
@@ -0,0 +1,13 @@
+OctoLinker is the easiest and best way to navigate between files and projects on GitHub. It turns language-specific statements like include, require or import into links. It supports a variety of languages such as JavaScript, Ruby, Go, PHP, JAVA and more. It works with package.json as well as with Gemfiles.
```

Comment:
It might be worth adding a full list of supported tools and languages at the bottom so that people searching for "npm" will find it